System implementation (#24731)
sympy/sympy: doc/src/modules/physics/mechanics/joints.rst

Joints Framework in Physics/Mechanics
=====================================

> :mod:`sympy.physics.mechanics` provides a joints framework. The
> :obj:`joints<sympy.physics.mechanics.joint>` themselves are used  been are used
> to define connections between bodies (:class:`~.Particle`,
> :class:`~.RigidBody`). This has been implemented such that it works well in
> combination with :class:`~.System` when forming the equations of motion. The
> main responsibility of the :obj:`joints<sympy.physics.mechanics.joint>` is to
> establish the kinematic relationships and do what we call "book-keeping" of
> single :obj:`joints<sympy.physics.mechanics.joint>`. The word "book-keeping" in
> this context means keeping track of the relationships that of that joint.

Reviewer: ```suggestion
:mod:`sympy.physics.mechanics` provides a joints framework. The
:obj:`joints<sympy.physics.mechanics.joint>` objects themselves are used
to define connections between bodies (:class:`~.Particle`,
:class:`~.RigidBody`). This has been implemented such that it works well in
combination with :class:`~.System` when forming the equations of motion. The
main responsibility of the :obj:`joints<sympy.physics.mechanics.joint>` is to
establish the kinematic relationships and do "book-keeping" for
individual :obj:`joints<sympy.physics.mechanics.joint>` objects. In this context, "book-keeping"
means keeping track of the relationships that of that joint.
```

Joints in Physics/Mechanics
===========================

> :mod:`joints<sympy.physics.mechanics.joint>` is create kinematic relationships
> between bodies (:class:`~.Particle`, :class:`~.RigidBody`). The general
> description of a joint is generally is shown in the image below.

Reviewer: ```suggestion
:mod:`joints<sympy.physics.mechanics.joint>` is used to create kinematic relationships
between bodies (:class:`~.Particle`, :class:`~.RigidBody`). The general
description of a joint is shown in the image below.
```

=====================================

:mod:`sympy.physics.mechanics` provides a joints framework. The

> :obj:`joint<sympy.physics.mechanics.joint>` objects themselves are used are used

Reviewer: ```suggestion
:obj:`joint<sympy.physics.mechanics.joint>` objects themselves are used
```

   >>> child.masscenter.vel(parent.frame)
   3*u*child_frame.y

System in Physics/Mechanics
===========================

> After defining the entire system you can use the :class:`~.System` to do a more
> general "book-keeping
> keep track of all the objects like joints and form the equations of motion. In
> this process the :class:`~.JointsMethod` only does the "book-keeping" of the joints. It uses

Reviewer: ```suggestion
After defining the entire system, you can use :class:`~.System` to do a more
general "book-keeping" that will
keep track of all the system's components (e.g. joints) and form the equations of motion. In
this process the :class:`~.JointsMethod` only does the "book-keeping" of the joints. It uses
```

Reviewer 2: Seems I forgot to finish this section
